test/CMakeLists.txt

```diff
@@ -1324,19 +1325,19 @@ foreach (coltype total tropo)
                                   tropomi_no2_${coltype}.nc ${IODA_CONV_COMP_TOL})
 endforeach()
 
-foreach (coltype total troposphere stratosphere)
```

Comment:
Is it worth the effort to make tempo and troponin consistent in using `troposphere` vs. `tropo`? This is just a suggestion.